### PR TITLE
feat: support http_target argument

### PR DESCRIPTION
**Pull request by svenlito**

### CODE REVIEW

File: versions.tf [hunk 1]
```diff
@@ -2,6 +2,6 @@ terraform {
   required_version = ">= 0.13.1"
 
   required_providers {
-    aws = ">= 3.19"
+    aws = ">= 3.44"
```
Comment (antonbabenko) [inline versions.tf:5]: It was added in 3.40 - https://github.com/hashicorp/terraform-provider-aws/blob/main/CHANGELOG.md#3400-may-13-2021

Comment (svenlito) [reply versions.tf:5]: @antonbabenko thank you, will update!

File: main.tf [hunk 2]
```diff
@@ -122,6 +122,17 @@ resource "aws_cloudwatch_event_target" "this" {
     }
   }
 
+  dynamic "http_target" {
+    for_each = lookup(each.value, "http_target", null) != null ? [
+      each.value.http_target
+    ] : []
+
+    content {
+      query_string_parameters = http_target.value.query_string_parameters
```
Comment (antonbabenko) [inline main.tf:131]: There are 3 arguments supported according to https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/cloudwatch_event_target

Also, all of them should be optional.

Comment (svenlito) [reply main.tf]: Clearly didn't pay attention there...

Comment (antonbabenko) [reply main.tf]: No problems! Looks good to me now :)

**Review by antonbabenko — APPROVED** [review]

**Comment by antonbabenko** [comment]
Should I release this one?

**Comment by svenlito** [comment]
> Should I release this one?

If you could, yes please.

**Comment by antonbabenko** [comment]
🎉  

[v1.4.0](https://github.com/terraform-aws-modules/terraform-aws-eventbridge/releases/v1.4.0) has been just released.

**Comment by ethompsy** [comment]
Any examples for using this functionality?

**Comment by svenlito** [comment]
> Any examples for using this functionality?

@ethompsy there is on in the [docs ](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/cloudwatch_event_target#example-api-gateway-target)

**Pull request merged**